src/app/components/jsx-helpers/jsx-helpers.jsx

```diff
@@ -66,12 +66,37 @@ export function LoaderPage({
     );
 }
 
+// Making scripts work, per https://stackoverflow.com/a/47614491/392102
+function activateScripts(el) {
+    const scripts = Array.from(el.querySelectorAll('script'));
+    const processOne = (() => {
+        if (scripts.length === 0) {
+            return;
+        }
+        const s = scripts.shift();
+        const newScript = document.createElement('script');
+        const p = (s.src) ? new Promise((resolve) => {
+            newScript.onload = resolve;
+        }) : Promise.resolve();
+
+        Array.from(s.attributes)
+            .forEach((a) => newScript.setAttribute(a.name, a.value));
+        newScript.appendChild(document.createTextNode(s.textContent));
+        newScript.async = false;
+        s.parentNode?.replaceChild(newScript, s);
+
+        p.then(processOne);
+    });
+
+    processOne();
+};
+
 export function RawHTML({Tag='div', html, embed=false, ...otherProps}) {
```

Comment:
I've used https://www.npmjs.com/package/loadjs for things like this before

Reply:
I think that's for a different purpose. This is to allow RawHTML with `<script>` pieces -- usually Wistia video players.

Reply:
hah, I thought you were inserting script tags to load js